src/ruby/yast/core_ext.rb

```diff
@@ -0,0 +1 @@
+require_relative "core_ext/string"
```

Comment:
well, I know this is active support way, another option is module way, when you create module and if you need then use
```
a = "abc"
a.extend AnsiCleaning
a.remove_ansi_sequences
```
Advantage of the second way is that string class is not so fat. Also it is less magic. Disadvantage is that you need another call.